usr/share/rear/lib/_input-output-functions.sh

```diff
@@ -791,6 +791,34 @@ preferably the whole debug information via 'rear -D $WORKFLOW'
 ===================="
 }
 
+# Error out with a deprecation info
+# first arg is feature keyword
+# everything else is a reason for the deprecation
+function ErrorIfDeprecated () {
```

Comment:
I was also wondering about that and chose the "If" form to signal that there is a decision that happens in the function.

Any other opinions on the function name @rear/contributors? Otherwise I'd like to keep the current name as it makes it clear that there is a conditional involved, whereas `BugError` or `DeprecationError` doesn't sound like there is any decision involved.

Reply:
'ErrorIfDeprecated' is OK for me,
it sounds only a bit confusing for my personal perception.
If hell broke loose because of this name whe can easily
replace it at any later time because it is an internal name
(in contrast to user config variable names).

By the way:
If we ever implement something as I suggested in
https://github.com/rear/rear/issues/2695
then the Error() function would actually
also behave as an ErrorIf...() function
under certain circumstances.